VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -113,13 +125,54 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         );
     }
 
-    getPrivateConsultationParticipants(): ParticipantListItem[] {
+    isJohConsultation(): boolean {
+        return this.roomLabel?.toLowerCase().includes('judgejohconsultationroom') ? true : false;
+    }
+
+    isFirstJohFromGroupMember(participant: any): boolean {
+        if (participant.hearing_role === HearingRole.PANEL_MEMBER)
+            return this.panelMembers[0].id === participant.id;
+        
+        if (participant.hearing_role === HearingRole.WINGER)
+            return this.wingers[0].id === participant.id;
+
+        if (participant.hearing_role === HearingRole.JUDGE)
+            return this.judge.id === participant.id;          
+    }
+
+    isLastJohFromGroupMember(participant: any): boolean {
```

Comment:
Looks like this is only used for applying a class? If so, it's unecessary, just use css.

Reply:
We need it to determine the participants order on a group